enterprise/api_client/discovery.py

```diff
@@ -116,10 +118,20 @@ def traverse_pagination(response, endpoint, content_filter_query, query_params):
 
         return results
 
-    def get_catalog_results(self, content_filter_query, query_params=None, traverse_pagination=False):
+    def _get_catalog_results_from_discovery(self, content_filter_query, query_params=None, traverse_pagination=False):
```

Comment:
Since we are using `_get_catalog_results_from_discovery` in tests can we remove underscore prefix from its name?